internal/databases/sqlserver/log_restore_handler.go

```diff
@@ -89,16 +110,24 @@ func restoreSingleLog(ctx context.Context,
 	logBackupName string,
 	dbname string,
 	fromname string,
-	stopAt time.Time) error {
+	stopAt time.Time,
+	lastPiece bool) error {
```

Comment:
why do we need this flag?

Reply:
Not to add WITH STOPAT to every log restore operation except the last one.